crd2pulumi-crontabs/README.md

```diff
@@ -0,0 +1,233 @@
+[![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
+
+# Generating CronTab CustomResources with `crd2pulumi`
+
+This example generates a strongly-typed CronTab CustomResource from the k8s CRD specified in `resourcedefinition.yaml` in TypeScript and Go. Afterwards, we'll use this generated code with Pulumi to deploy the CRD and create an instance. For more documentation on `crd2pulumi`, check out the [project's GitHub page](https://github.com/pulumi/pulumi-kubernetes/tree/master/provider/cmd/crd2pulumi).
```

Comment:
Nit:

> resourcedefinition.yaml

Should we rename this file `crd.yaml` (like in the blog post) or even `crontabs.yaml`?